code/modules/mob/living/carbon/human/species/station/machine_subspecies.dm

```diff
@@ -223,13 +224,9 @@
 			playsound(H.loc, 'sound/machines/buzz-two.ogg', 100, 0)
 	return 0
 
-/datum/species/machine/handle_death(var/mob/living/carbon/human/H)
+/datum/species/machine/terminator/handle_death(var/mob/living/carbon/human/H)
 	..()
-	spawn(10)
-		playsound(H.loc, 'sound/items/countdown.ogg', 125, 1)
-		spawn(20)
-			explosion(H, -1, -1, 5)
-			H.gib()
-
-/datum/species/machine/sanitize_name(var/new_name)
-	return sanitizeName(new_name, allow_numbers = 1)
\ No newline at end of file
+	playsound(H.loc, 'sound/items/countdown.ogg', 125, 1)
+	spawn(15)
+		explosion(src, -1, 1, 5)
```

Comment:
`explosion(src, -1, 1, 5)` No bueno. `src` is the datum which is at coordinates 0 0 0. You probably want `explosion(H, -1, 1, 5)` or `explosion(H.loc, -1, 1, 5)`.